modules/items/server/policies/items.server.policy.js

```diff
@@ -33,6 +33,11 @@ exports.invokeRolesPolicies = function () {
     {
       resources: '/api/modules',
       permissions: '*'
+    },
```

Comment:
Is this necessary?

Reply:
Yeah I've had it removed when I first started this and the link wouldnt show on the items list.

Reply:
It's not necessary, I just tried without it. Maybe there were other changes that did mess things up but idk.